panic runtime error

### Windows

```
2018-08-01T08:13:06.450+0900` [DEBUG] plugin: starting plugin: path=C:\dkron_0.10.4_windows_amd64\dkron-processor-files.exe args=[C:\dkron_0.10.4_windows_amd64\dkron-processor-files.exe]
2018-08-01T08:13:06.513+0900 [DEBUG] plugin: waiting for RPC address: path=C:\dkron_0.10.4_windows_amd64\dkron-processor-files.exe
2018-08-01T08:13:06.596+0900 [DEBUG] plugin.dkron-processor-files.exe: plugin address: timestamp=2018-08-01T08:13:06.540+0900 address=127.0.0.1:10004 network=tcp
2018-08-01T08:13:06.618+0900 [DEBUG] plugin: starting plugin: path=C:\dkron_0.10.4_windows_amd64\dkron-processor-log.exe args=[C:\dkron_0.10.4_windows_amd64\dkron-processor-log.exe]
2018-08-01T08:13:06.622+0900 [DEBUG] plugin: waiting for RPC address: path=C:\dkron_0.10.4_windows_amd64\dkron-processor-log.exe
2018-08-01T08:13:06.710+0900 [DEBUG] plugin.dkron-processor-log.exe: plugin address: timestamp=2018-08-01T08:13:06.648+0900 address=127.0.0.1:10005 network=tcp
2018-08-01T08:13:06.726+0900 [DEBUG] plugin: starting plugin: path=C:\dkron_0.10.4_windows_amd64\dkron-processor-syslog.exe args=[C:\dkron_0.10.4_windows_amd64\dkron-processor-syslog.exe]
2018-08-01T08:13:06.731+0900 [DEBUG] plugin: waiting for RPC address: path=C:\dkron_0.10.4_windows_amd64\dkron-processor-syslog.exe
2018-08-01T08:13:06.818+0900 [DEBUG] plugin.dkron-processor-syslog.exe: plugin address: timestamp=2018-08-01T08:13:06.763+0900 network=tcp address=127.0.0.1:10006
2018-08-01T08:13:06.839+0900 [DEBUG] plugin: starting plugin: path=C:\dkron_0.10.4_windows_amd64\dkron-executor-shell.exe args=[C:\dkron_0.10.4_windows_amd64\dkron-executor-shell.exe]
2018-08-01T08:13:06.844+0900 [DEBUG] plugin: waiting for RPC address: path=C:\dkron_0.10.4_windows_amd64\dkron-executor-shell.exe
2018-08-01T08:13:06.927+0900 [DEBUG] plugin.dkron-executor-shell.exe: plugin address: timestamp=2018-08-01T08:13:06.871+0900 address=127.0.0.1:10007 network=tcp
INFO[2018-08-01T08:13:06+09:00] agent: Dkron agent starting                   node=Windows
INFO[2018-08-01T08:13:06+09:00] agent: joining: [192.168.0.4] replay: true    node=Windows
INFO[2018-08-01T08:13:06+09:00] agent: joined: 1 nodes                        node=Windows
INFO[2018-08-01T08:13:06+09:00] agent: Listen for events                      node=Windows
DEBU[2018-08-01T08:13:07+09:00] agent: Received event                         event=member-join node=Windows
DEBU[2018-08-01T08:13:07+09:00] agent: Member event                           event=member-join member=Windows node=Windows
DEBU[2018-08-01T08:13:07+09:00] agent: Member event                           event=member-join member=Linux node=Windows
DEBU[2018-08-01T08:13:59+09:00] agent: Received event                         event="query: run:job" node=Windows
DEBU[2018-08-01T08:13:59+09:00] agent: Running job                            at=10 node=Windows payload="{\"execution\":{\"job_name\":\"ok_windows\",\"started_at\":\"0001-01-01T00:00:00Z\",\"finished_at\":\"0001-01-01T00:00:00Z\",\"group\":1533078839965072800,\"attempt\":1},\"rpc_addr\":\"192.168.0.4:6868\"}" query="run:job"
INFO[2018-08-01T08:13:59+09:00] agent: Starting job                           job=ok_windows node=Windows
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0xc220e6]

goroutine 94 [running]:
github.com/victorcoder/dkron/dkron.(*Agent).eventLoop(0xc04218bb80)
        /Users/victorcoder/src/github.com/victorcoder/dkron/dkron/agent.go:445 +0x13c6
created by github.com/victorcoder/dkron/dkron.(*Agent).Start
        /Users/victorcoder/src/github.com/victorcoder/dkron/dkron/agent.go:93 +0x187
```

--- a/dkron/agent.go
+++ b/dkron/agent.go
@@ -90,6 +90,15 @@ func (a *Agent) Start() error {
 	if a.config.Server {
 		a.StartServer()
 	}
+
+	if a.GRPCClient == nil {
+		a.GRPCClient = NewGRPCClient(nil)
+	}
+
+	if err := a.SetTags(a.config.Tags); err != nil {
+		log.WithError(err).Fatal("agent: Error setting RPC config tags")
+	}
+
 	go a.eventLoop()
 	a.ready = true
 
@@ -298,13 +307,6 @@ func (a *Agent) StartServer() {
 		log.WithError(err).Fatal("agent: RPC server failed to start")
 	}
 
-	if a.GRPCClient == nil {
-		a.GRPCClient = NewGRPCClient(nil)
-	}
-
-	if err := a.SetTags(a.config.Tags); err != nil {
-		log.WithError(err).Fatal("agent: Error setting RPC config tags")
-	}
 	a.participate()
 }
 

